### PR TITLE
add doc for load-based replica read variable

### PR DESCRIPTION
**Pull request by you06**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add the doc for `tidb_load_based_replica_read_threshold`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): pingcap/tidb#41664

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [x] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -2141,6 +2141,14 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 类型：字符串
 - 这个变量是一个只读变量，用于获取当前会话中最后一个 `PLAN REPLAYER dump` 的结果。
 
+### `tidb_load_based_replica_read_threshold` <span class="version-mark">从 v7.0.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
+- 默认值："0s"
```
Comment (Oreoxmt) [inline system-variables.md:2147]:
```suggestion
- 默认值：`"0s"`
```

File: system-variables.md [hunk 2]
```diff
@@ -2141,6 +2141,14 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
+- 范围：`[0s, 1h]`
+- 类型：字符串
+- 这个变量用来设置 load-based replica read 的触发阈值。当 leader 的预估排队时间高于阈值时，TiDB 会优先从 follower 读取数据。格式为时间，例如 `100ms` 或 `1s`。
```
Comment (cfzjywxk) [inline system-variables.md:2150]: > 当 leader 的预估排队时间高于阈值时

这个变量具体的用法或者排队的含义可能要单独分一个页面介绍下？

[the remark this answers is not in the document]
Comment (you06) [reply system-variables.md:2150]: 后面说的是 "follower 节点"，前面也改成 "当 leader 节点的..." ?

**Review by cfzjywxk — APPROVED** [review]

File: system-variables.md (hunk 2 above)
Comment (you06) [reply system-variables.md:2150]: 感觉有回表优化的最佳实践之后可以单独分一个页面来讲讲。

File: system-variables.md [hunk 3]
```diff
@@ -2141,6 +2141,14 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 类型：字符串
 - 这个变量是一个只读变量，用于获取当前会话中最后一个 `PLAN REPLAYER dump` 的结果。
 
+### `tidb_load_based_replica_read_threshold` <span class="version-mark">从 v7.0.0 版本开始引入</span>
+
+- 作用域：SESSION | GLOBAL
```
Comment (Oreoxmt) [inline system-variables.md:2146]:
```suggestion
- 作用域：SESSION | GLOBAL
- 是否持久化到集群：是/否
```

**Review by ran-huang — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: system-variables.md [hunk 4]
```diff
@@ -2141,6 +2141,15 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 类型：字符串
 - 这个变量是一个只读变量，用于获取当前会话中最后一个 `PLAN REPLAYER dump` 的结果。
 
+### `tidb_load_based_replica_read_threshold` <span class="version-mark">从 v7.0.0 版本开始引入</span>
+
```
Comment (ran-huang) [inline system-variables.md:2145]:
```suggestion

> **警告：**
>
> 当前版本中该变量控制的功能尚未完全生效，请保留默认值。

```

**Pull request merged**